Resources/Prototypes/Loadouts/Jobs/Medical/traitRestrictedGear.yml

```
@@ -0,0 +1,462 @@
+  category: JobsMedicalParamedic
+  cost: 3
+  requirements:
+    - !type:CharacterTraitRequirement
+      traits:
+        - RestrictedGear
+    - !type:CharacterJobRequirement
+      inverted: true
+      jobs:
+        - Paramedic
+  items:
+    - Portafib
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)